pages/dash_download_component_app.py

```diff
@@ -5,22 +5,23 @@
 
 dash.register_page(
     __name__,
-    description="This app let's the user download datatable data as excel or csv files, using the dcc.Download component and a dropdown.",
+    description="This app let's the user download datatable data as excel or csv files, using the dcc.Download component and a dropdown. It has a top-bottom layout and a regular-callback.",
 )
 
 filename = __name__.split("pages.")[1]
 
 # any notes will be displayed below the code-and-show page in a dcc.Markdown component
 notes = """
 #### Dash Components in App:
-- [DataTable component](https://dash.plotly.com/datatable)
-- [Dropdown component](https://dash.plotly.com/dash-core-components/dropdown)
-- [Download component](https://dash.plotly.com/dash-core-components/download)
+- [DataTable](https://dash.plotly.com/datatable)
+- [Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
+- [Download](https://dash.plotly.com/dash-core-components/download)
 
 #### Community Components:
 
 Dash Bootstrap Components
-- [Button component](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/button/ "This component was made by the community and not officially maintained by Plotly.")
+- [Layout](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/layout/ "This component was made by the community and not officially maintained by Plotly.")
```

Comment:
Let's take out this link to the layout component. I think I put that there by mistake, but it's too repetitive because every app with dash bootstrap container will have a layout section.